content/docs/command-reference/repro.md

```diff
@@ -105,15 +126,21 @@ up-to-date and only execute the final stage.
   recursive search for changed dependencies. Multiple stages are executed
   (non-recursively) if multiple stage names are given as `targets`.
 
-- `-R`, `--recursive` - determines the stages to reproduce by searching each
-  target directory and its subdirectories for stages (in `dvc.yaml`) to inspect.
-  If there are no directories among the targets, this option is ignored.
+- `-R`, `--recursive` - looks for `dvc.yaml` files to reproduce in any
+  directories given as `targets`, and in their subdirectories. If there are no
+  directories among the targets, this option has no effect.
 
 - `--no-commit` - do not store the outputs of this execution in the cache
   (`dvc.yaml` and `dvc.lock` are still created or updated); useful to avoid
   caching unnecessary data when exploring different data or stages. Use
   `dvc commit` to finish the operation.
 
+- `-p`, `--pipeline` - reproduce the entire pipelines that the `targets` belong
```

Comment:
not relevant? (I know it might be painful, but otherwise we iterate too long on all these changes). Especially when team is involved, let's keep it to the point and iterate fast please

Reply:
So the reason some options are changed here is because they involve the `targets` argument.

TBH I'm confused with the idea that this Pr is about glob when it was always about targets even more than about --glob. But would you like me to split into 2 PRs?

Reply:
but you do at least a few changes at once - you move two options, you also introduce some changes to one of them. Also the change is about `dvc dag`, not even dvc repro in this case

Reply:
OK here I was confused indeed, not about `targets` per se. Reverted move.